### PR TITLE
Updated extensionsGallery.json to have 1.0.0 arc and azcli

### PR DESCRIPTION
**Pull request by candiceye**
Update extensionsGallery.json to have arc and azcli be version 1.0.0 for Feb release.

### CODE REVIEW

**Comment by erpett** [comment]
Where are the extension files that we need to upload?

File: extensionsGallery.json [hunk 1]
```diff
@@ -3316,14 +3316,14 @@
...
-							"lastUpdated": "2/15/2022",
+							"version": "1.0.0",
+							"lastUpdated": "2/22/2022",
 							"assetUri": "",
 							"fallbackAssetUri": "fallbackAssetUri",
 							"files": [
 								{
 									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/arc/arc-0.12.0.vsix"
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/arc/arc-1.0.0.vsix"
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:3326]: Make sure you update the engine versions here as well

**Comment by candiceye** [comment]
Version bump PRs are here:
main: https://github.com/microsoft/azuredatastudio/pull/18516
release/1.35 branch: https://github.com/microsoft/azuredatastudio/pull/18526

**Comment by Charles-Gagnon** [comment]
Let's use the release candidate build instead : https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=142184&view=results

**Comment by erpett** [comment]
To confirm - this is ready for me to take action on?

**Comment by Charles-Gagnon** [comment]
Yup, once the extension owner provides you with the VSIXs you should upload them and then if the validate-galleries step passes you're good to merge in. (no need to wait for me or anyone else to sign off for RC1, you'll be in charge of that branch)

**Comment by candiceye** [comment]
@erpett Yep, ready for you to take action on. Please use the extension vsix files in the link Charles provided https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=142184&view=results

**Pull request merged**